spec/provider_core_spec.rb

```diff
@@ -17,28 +17,28 @@
   context '#format' do
     subject { described_class.new(mock('config'), mock('api')) }
 
-    it 'fails if cannot fit' do
-      expect do
-        subject.format [['x' * 21]], 0, 0, 20
-      end.to raise_error ArgumentError, /No code fits/
+    it 'does not fail if cannot fit' do
+      expect(
+        subject.format([['x' * 41]], 0, 0, 20)
+      ).to include('rubocop:disable Metrics/LineLength')
     end
 
-    it 'fails if cannot fit any' do
-      expect do
-        subject.format [['x' * 21], ['y' * 21], ['z' * 30]], 0, 0, 20
-      end.to raise_error ArgumentError, /No code fits/
+    it 'does not fail if cannot fit any' do
+      expect(
+        subject.format([['x' * 31], ['y' * 31], ['z' * 30]], 0, 0, 20)
+      ).to include 'rubocop:disable Metrics/LineLength'
     end
 
-    it 'fits 80 chars' do
-      subject.format [['x' * 80]]
+    it 'fits 100 chars' do
+      subject.format [['x' * 100]]
```

Comment:
Now that we don't raise an exception, you should expect this not to include a line length disable.  Same with all other tests that used to depend on crashes to fail.

Reply:
These tests should verify that format works properly, but I don't think that includes the "LineLength" portion.

Check out the changes. I think the better way to verify that if < 80, the output is exactly as expected, where > 80 includes LineLength.

Reply:
Also, pushes on Github are being weird so I'll probably have to re-push + fiddle with the branch in a bit.